app/views/index/index.html.erb

```diff
@@ -25,10 +25,13 @@
 
       <% if EmailValidator.email_is_allowed_advanced?(@email) %>
       <h3 class="govuk-heading-s">Request a new AWS user</h3>
-      <p>GDS manages a number of AWS accounts. Users for these accounts are managed centrally by reliability engineering with a base GDS account: <em>gds-users</em>.</p>
+      <p>GDS manages a number of AWS accounts. Users for these accounts are managed centrally by Engineering Enablement with a base GDS account: <em>gds-users</em>.</p>
```

Comment:
We can sidestep questions about who manages gds-users by saying something like:

"There are a number of ways of accessing your AWS accounts, but the recommended way is to use AWS's [cross account access pattern](https://gds-way.digital.cabinet-office.gov.uk/manuals/working-with-aws-accounts.html#cross-account-access). To support this, we maintain a set of centrally managed user identities in a base account called `gds-users`.